develop/pages/home/index.html

```diff
@@ -24,47 +30,95 @@
         </div>
         <div class="row">
             <div class="col-md-3 col-lg-2">
-                <!--<nav class="AppNavigation">
-                    <ul class="list-group">
-                        <li class="list-group-item"><a href="KO-templates/KO-categories.html">Categories</a></li>
-                        <li class="list-group-item"><a href="KO-templates/KO-products.html">Products</a></li>
+                <div class="AppNavigation" data-bind="visible: DataModel.prototype.isCatReady() === true">
```

Comment:
When property is boolean then you don't have compare it with true/false. just use isCatReady() or !isCatReady()